system.go

```diff
@@ -43,3 +53,11 @@ func (g *Glutton) startMonitor(quit chan struct{}) {
 		}
 	}()
 }
+
+func isCommandAvailable(name string) bool {
+	cmd := exec.Command("/bin/sh", "-c", "command -v "+name)
```

Comment:
I think this line needs gofmt

Reply:
@glaslos I already applied gofmt on the command line :) This line still remained unmodified